Kernel/VM/PhysicalRegion.cpp

```diff
@@ -0,0 +1,26 @@
+#include <AK/Retained.h>
+#include <Kernel/Assertions.h>
+#include <Kernel/VM/MemoryManager.h>
+#include <Kernel/VM/PhysicalRegion.h>
+
+Retained<PhysicalRegion> PhysicalRegion::create(PhysicalAddress lower, PhysicalAddress upper)
+{
+    return adopt(*new PhysicalRegion(lower, upper));
+}
+
+PhysicalRegion::PhysicalRegion(PhysicalAddress lower, PhysicalAddress upper)
+    : m_lower(lower), m_upper(upper), m_next(lower)
```

Comment:
If you run clang-format on this code I think it will break this into three lines. Let’s do whatever clang-format does :)